app/src/main/res/values-lt/strings.xml

```diff
@@ -0,0 +1,328 @@
+    <string name="form_create_meal_description">Aprašymas</string>
+    <string name="form_create_meal_duration">Gaminimo trukmė</string>
+    <string name="form_create_meal_ingredients">Ingredientai</string>
+    <string name="form_create_meal_preparation">Paruošimas</string>
+    <string name="form_create_meal_title">Pavadinimas</string>
+    <string name="form_survey_toast_error">Nepavyko pateikti</string>
+    <string name="friends_button_add_friend">Pridėti draugą</string>
+    <string name="friends_button_add_friend_input_label">Draugo paštas</string>
+    <string name="friends_button_requests">Užklausos</string>
+    <string name="friends_button_request_dialog_empty_received">Nėra gautų užklausų</string>
+    <string name="friends_button_request_dialog_empty_sent">Nėra išsiųstų užklausų</string>
+    <string name="friends_button_request_dialog_tab_type_received">Gautos</string>
+    <string name="friends_button_request_dialog_tab_type_sent">Išsiųstos</string>
+    <string name="friends_button_request_dialog_title">Draugų užklausos</string>
+    <string name="friends_error_already_friend">%1$s jau yra jūsų draugas</string>
+    <string name="friends_error_already_sent_request">Draugavimo užklausa jau išsiųsta pas %1$s</string>
+    <string name="friends_error_input_empty">Įveskite el. paštą</string>
+    <string name="friends_error_invite_self">Jūs visada liksite savo geriausiu draugu</string>
+    <string name="friends_error_not_exists">Naudotojas su paštu %1$s neegzistuoja</string>
+    <string name="friends_error_not_found">Draugas nerastas</string>
+    <string name="friends_size">Viso draugų - %1$s</string>
+    <string name="friends_sort_by">Rūšiuoti pagal</string>
+    <string name="friends_sort_type_distance">Atstumas</string>
+    <string name="friends_sort_type_experience">Patirtis</string>
+    <string name="friends_sort_type_goals">Siekiai</string>
+    <string name="friends_sort_type_points">Taškai</string>
+    <string name="friends_sort_type_steps">Žingsniai</string>
+    <string name="friends_sort_type_tasks">Užduotys</string>
+    <string name="friends_success_already_received_request">Tiek jūs, tiek jūsų draugas kvietėtės vienas kitą. Užklausos priimtos</string>
+    <string name="friends_success_remove">Draugas sėkmingai panaikintas</string>
+    <string name="friends_success_request_accepted">Draugo užklausa priimta</string>
+    <string name="friends_success_request_canceled">Draugavimo užklausa atšaukta</string>
+    <string name="friends_success_request_denied">Draugo užklausa atmesta</string>
+    <string name="friends_success_sent">Draugavimo užklausa išsiųsta</string>
+    <string name="friend_7_days_activity">7 dienų aktyvumas</string>
+    <string name="friend_button_remove">Panaikinti draugą</string>
+    <string name="friend_button_remove_confirmation_description">Jeigu draugas bus panaikintas, turėsite jį pakviesti iš naujo į draugus.</string>
```

Comment:
Maybe just "Jeigu draugas bus panaikintas, turėsite jį pakviesti iš naujo" would suffice? The last "į draugus" sounds a bit odd.